vsintegration/src/vs/FsPkgs/FSharp.VS.FSI/fsiCommands.vsct

```diff
@@ -119,7 +119,7 @@
       <Button guid ="guidInteractive" id ="cmdidDebugSelection" priority ="0x106" type ="Button">
         <Parent guid="guidSHLMainMenu" id="IDG_VS_CODEWIN_LANGUAGE"/>
         <Strings>
-          <ButtonText>Debug in F# Interactive</ButtonText>
+          <ButtonText>Debug In Interactive</ButtonText>
```

Comment:
I don't know, I guess that's just the preferred style.  Seems pretty consistent to capitalize even the small words ("Find All References", "Go To Definition", etc). "Search with Pattern" is maybe an oversight.

![image](https://cloud.githubusercontent.com/assets/5943573/7090326/19d4697c-df57-11e4-984c-e35fffc2d5f4.png)